_posts/2023-02-15-my-first-patch-set.md

```diff
@@ -0,0 +1,153 @@
+---
+layout: post
+title:  "My First Patch Set"
+date:   2023-02-15 11:57:26 -0300
+categories: jekyll update
+---
+
+Hello everyone! In this Post i will describe my experience writing my first patch set to the Linux
+Kernel, more specifically in the [Direct Rendering Manager (DRM)][wikipedia-drm] subsystem.
+
+## Why I'm Doing This
```

Comment:
It would be nice to hear a little about your engagement with open-source. I mean, why do you use Linux in the first place? Why have you contributed to projects like LLVM previously? This kind of stuff.